index.html

```diff
@@ -16,14 +16,18 @@
     <div class="collapse navbar-collapse" id="navbarNav">
       <ul class="navbar-nav mx-auto text-center">
         <li class="nav-item">
-          <a class="menu__link menu__link--color nav-link" href="#about-me">About me<span class="sr-only">(current)</span></a>
+          <a class="menu__link menu__link--color nav-link" href="#">Home</a>
```

Comment:
atrybut href z wartością # spowoduje przeładowanie strony - co może spowodować usunięcie danych użytkownika z formularza. Może zastosuj technikę kotwicy jak to zrobileś dla innych linków menu?